src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(), ColorSupplier.random.nextInt(101));
```

Comment:
The method getRandomBall() should not use static methods or fields from ColorSupplier. Instead, ensure that random number generation is handled within the Lottery class or through an instance method of ColorSupplier.